cli/mfg.c

```diff
@@ -404,12 +404,50 @@ static int image_select(int argc, char **argv)
 	return ret;
 }
 
+static int boot_resume(int argc, char **argv)
+{
+	const char *desc = "Resume device boot process (BL1 and BL2 only)";
```

Comment:
Please define "Resume" as this is not at all clear.

Reply:
More descriptions added. Please take a look.